pkg/kubeadm/master/manifests.go

```diff
@@ -213,6 +214,7 @@ func getComponentCommand(component string, s *kubeadmapi.KubeadmConfig) (command
 
 	if component == controllerManager && s.InitFlags.CloudProvider != "" {
 		command = append(command, "--cloud-provider="+s.InitFlags.CloudProvider)
+		command = append(command, DefaultCloudConfig)
```

Comment:
Ah, yeah, you're right. Will fix that.
Should the option be present for both apiserver and the cm?